backend/functions/handlers/events.js

```diff
@@ -1,26 +1,20 @@
 const { db } = require("../util/admin");
-const gcal = require('../util/googlecal.js');
-const Timestamp = require('firebase-firestore-timestamp');
-var moment = require('moment-timezone');
-console.log(moment.tz.guess());
+const gcal = require("../util/googlecal.js");
 
 
 exports.getAllEvents = (req, res) => {
-  
   db.collection("events")
     .get()
     .then(doc => {
       let events = [];
       doc.forEach(data => {
-        let zoneInfo=moment.tz.guess();
-         let time=moment(data.data().time).tz(zoneInfo);
-         let timezone=time.format();
         events.push({
           eventId: data.id,
           imageUrl: data.data().imageUrl,
           name: data.data().name,
           organizer: data.data().organizer,
-          time: timezone,
+          startTime: data.data().startTime,
+          endTime: data.data().endTime,
```

Comment:
Is it possible to change these to start and end?

```suggestion
          start: data.data().startTime,
          end: data.data().endTime,
```

Reply:
@Germain-L  has implemented the mobile application using these fields. He's releasing the APK with these fields, so if we change them now, his application won't work at all. Perhaps later, when he's releasing a new patch.